mpap/src/crawler.py

```diff
@@ -0,0 +1,154 @@
+
+
+BASE_URL_MEMBROS_ATIVOS = 'https://www.mpap.mp.br/transparencia/index.php?pg=consulta_folha_membros_ativos'
+BASE_URL_VERBAS_INDENIZATORIAS = 'https://www.mpap.mp.br/transparencia/index.php?pg=consulta_verbas_indenizatorias'
+
+FLAG = ['remuneracao', 'verbas-indenizatorias']
+# Usado para fazer as comparações
+REMUNERACAO = 'remuneracao'
+VERBAS_INDENIZATORIAS = 'verbas-indenizatorias'
+
+
+def crawl(month, year, driver_path, output_path):
+    files = []
+
+    pathlib.Path(output_path).mkdir(exist_ok=True)
+    driver = setup_driver(driver_path, output_path)
+
+    sleep(4)
+    # Roda duas vezes, uma para pegar a planilha de remuneração, e outra para as de verbas indenizatorias
+    for flag in FLAG:
+        if flag == REMUNERACAO:
+            # Carrega a pagina de remuneração
+            driver.get(BASE_URL_MEMBROS_ATIVOS)
+            sleep(5)
+            file_path = download(month, year, output_path, driver, flag)
+            files.append(file_path)
+
+        elif flag == VERBAS_INDENIZATORIAS:
+            # Carrega a pagina de verbas indenizatorias
+            driver.get(BASE_URL_VERBAS_INDENIZATORIAS)
+            sleep(5)
+            file_path = download(month, year, output_path, driver, flag)
+            files.append(file_path)
+
+    driver.quit()
+    return files
+
+
+def download(month, year, output_path, driver, flag):
+    # Como o id de ano muda nas páginas, usei o try except para se der erro em um é o outro.
+    try:
+        current_year = driver.find_element(By.XPATH, '//*[@id="ano"]')
+    except:
+        current_year = driver.find_element(By.XPATH, '//*[@id="ano_verbas"]')
+
+    # Esse Select server para pegar as tags select, onde posso passar o valor diretamente,
+    # para pegar qual opção eu quero.
+    select_year = Select(current_year)
+    select_year.select_by_value(year)
+
+    sleep(4)
+    current_month = driver.find_element(By.XPATH, '//*[@id="mes"]')
+    select_month = Select(current_month)
+    # Como os numeros passados quando convertidos para string ficam com um "0" na frente,
+    # uso isso para remover, e pegar corretamente o valor da tag.
+    select_month.select_by_value(month.replace('0',''))
+
+    sleep(2)
+    button_enviar = driver.find_element(By.XPATH, '//*[@id="enviar"]')
+    button_enviar.click()
+
+    # Formating the filename
+    sleep(6)
+    file_name = ''
+    # A página de remuneração está com problema para exportar a planilha, então separei em duas funções,
+    # uma pega a pagina e com o beautifulsoup e outros, transforma em uma documento csv.
+    if flag == REMUNERACAO:
+        file_name = download_remuneracao(driver, output_path, month, year, flag)
+    elif flag == VERBAS_INDENIZATORIAS:
+        file_name = download_indenizacao(driver, output_path, month, year, flag)
+
+    return file_name
+
+# Usado para transformar a pagina html com a planilha, em doc csv, com os valores.
+def download_remuneracao(driver, output_path, month, year, flag):
```

Comment:
```suggestion
def download_remuneracao(driver, output_path, month, year):
```

Com a sugestão que fiz abaixo a flag não é mais necessária